src/vmspawn/vmspawn.c

```diff
@@ -57,6 +61,7 @@ STATIC_DESTRUCTOR_REGISTER(arg_image, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_machine, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_qemu_smp, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_qemu_kernel, freep);
+STATIC_DESTRUCTOR_REGISTER(arg_qemu_initrd, freep);
```

Comment:
shouldn't this be an strv? i.e. kernels are fine with multiple initrds. does qemu support that? if so, then we should do that too. After all we already use multiple initrds for passing through creds, and sysexts and all that stuff. seems odd to not support this here if we make use of it already and qemu supports it too

Reply:
I've implemented this now, I'm not 100% sure the arguments to qemu are correct but it does seem to boot when I pass it 2 initrds.
I've implemented it by -initrd then all of the initrds after that as this was how I interpreted the qemu man page.
Fixes are in 37e75950ff.